ecocode-rules-specifications/src/main/rules/EC1/EC1.json

```diff
@@ -3,15 +3,10 @@
   "type": "CODE_SMELL",
   "status": "ready",
   "remediation": {
-    "func": "Constant\/Issue",
```

Comment:
did you test your modification in a local SonarQube to check if the description is well displayed ?

Reply:
Those func attribute changes were reverted.